optimize/self-managed/optimize-deployment/migration-update/3.10-to-3.11_8.3.md

```diff
@@ -17,6 +17,12 @@ Here you will find information about:
 - Changes in behavior (for example, due to a new feature)
 - Changes in translation resources
 
+## Known issues
+
+## Migration of Camunda 7 data for 3.11.0, 3.11.1, and 3.11.2
+
+Under some circumstances, the migration of Camunda 7 data for versions 3.11.0, 3.11.1, and 3.11.2 can cause issues with the tenant selection of report definitions and collection scopes. This only occurs if data is present in Elasticsearch with the value `zeebe` in the datasource field, which can happen if the `onboarding` dataset set is used in Optimize, for example. To avoid this issue, we recommend Camunda 7 users skip 3.11.0, 3.11.1, and 3.11.2, and instead migrate straight to Optimize version 3.11.3.
```

Comment:
Do you think we should also include here that the customer onboarding dataset should not be used in C7 environments, and maybe a link to some config for how to disable it?

Reply:
Hm not sure, if they already have it then disabling the config isn't gonna remove it right? I would tend towards not complicating things, though I would like to follow up with checking if this was a normal use case to have the onboarding data in C7 and if so, whether we can adjust the set to not include the "zeebe" datasource so we dont have confusing data in the future.

Reply:
sure 👍 
I think this would be a good post-mortem topic. Re the zeebe datasource, this is needed for the dataset to work in SaaS, so I am not sure how changeable this would be for C7 customers